demos/supabase/README.md

```diff
@@ -4,26 +4,35 @@ This is demonstration of running Supabase locally in tandem with our [React Supa
 
 ## Getting Started
 
-Follow the [instructions](https://supabase.com/docs/guides/cli/getting-started) for configuring Supabase locally.
+Make sure you are using the latest version of the Supabase CLI. If you don't have the Supabase CLI installed, follow the [instructions](https://supabase.com/docs/guides/local-development/cli/getting-started#installing-the-supabase-cli). 
 
-Copy the environment variables template file
+Then, copy the environment variables template file
 
 ```bash
 cp .env.template .env
 ```
 
-Start the Supabase project
+Start the Supabase project using the setup script (this automatically generates asymmetric signing keys):
 
 ```bash
+chmod +x setup.sh
+./setup.sh
+```
+
+Alternatively, you can run the steps manually:
+
+```bash
+supabase gen signing-key --algorithm ES256 --append
+
 supabase start
 ```
 
-Once started the console will contain details for the Supabase services. Apply the `anon key` and `JWT secret` to the `.env` file.
+Once started the console will contain details for the Supabase services. Apply the `Publishable authentication key` to the `.env` file in the `SUPABASE_PUBLISHABLE_KEY` variable.
 
-Note: Default Supabase projects expose the Postgres DB on a high port number. We currently only support ports in the range of `port >= 1024 && port <= 49151`. This demo configures Postgres on port `5433`.
+> **Note:** This demo uses Supabase's new asymmetric JWT signing keys (ES256). PowerSync is compatible with these keys and will automatically fetch the public key from Supabase's JWKS endpoint. You no longer need to configure `PS_SUPABASE_JWT_SECRET`.
```

Comment:
Should we be mentioning the `PS_SUPABASE_JWT_SECRET` here at all if it technically does not exist anymore?

Reply:
We should be able to remove `PS_SUPABASE_JWT_SECRET` from the `docker-compose.yaml` file also.